tasks/prepare-binaries.js

```diff
@@ -104,36 +114,60 @@ async function main (platforms) {
     fsExtra.ensureDirSync(chromiumDir)
   }
 
-  // get browser
-  await getBrowsers(platforms)
-
-  // using pkg create the binary for asciidoctor-web-pdf
-  await createPackage(platforms)
-
+  if (isDryrun()) {
```

Comment:
If the goal is to make sure that the `archive` function works as expected then I think it would be better to write a test.

Reply:
Yes agreed, but right now I'm clueless why downloading takes endless time. It's not due to my Internet bandwith for sure. If you are unhappy with it I drop this change.